### ISSUE TITLE
Data entry forms: testing and merging latest changes

### ISSUE THREAD

**Issue by isedwards** [opencdms/Climsoft#535]
There are a few steps that are required before the latest updates to the data entry forms can be tested and merged.

@Patowhiz @maxwellfundi - could you work through this list and let us know when the PR is ready for testing?
- [ ] Check that all of the changes in Ivan's pull request (#510) are now included in Patrick's #528 and, if so, close Ivan's PR without merging
- [ ] Check whether any other work on the [dev](https://github.com/opencdms/Climsoft/tree/dev) branch should be part of the PR to be merged to master and, if so, add these to #528
- [ ] Locally, on your own machines, checkout the latest version of master and try merging #528 into your local version and test this to check that everything works as expected
- [ ] Create a new pull request for #528 to be merged to master (instead of dev)
- [ ] After #528 has been merged to master, raise a new issue to:
  - Create a new branch for `version_5` from master (that includes work from #528)
  - Copy any work from the old `dev` branch that is not intended for version 4 into the `version_5` branch
  - Confirm that `dev` can be removed

**Comment by mhabimana** [comment]
@Patowhiz @maxwellfundi :Any update or progress on this?,we need to finalise this and move forward.

**Comment by maxwellfundi** [comment]
@mhabimana yeah @Patowhiz  will be making the PR to master soon. All that we need is already on #528 because we combined them together.

**Comment by Patowhiz** [comment]
@mhabimana I'm finalising on this. We will be making the PR

**Comment by mhabimana** [comment]
@Patowhiz :Thanks,well noted.

**Comment by mhabimana** [comment]
@Patowhiz: We are still waiting your new PR,good to have this ready before the end of the month.

**Comment by mhabimana** [comment]
@patowhiz: We are still waiting your new PR to merge this with #534

**Comment by Patowhiz** [comment]
We now have all the changes committed. The only thing remaining is on solving the conflicts, we should be able to do this quite quickly.

**Comment by mhabimana** [comment]
@patowhiz - Great,well noted.
